tests/factories/PacbioLibraryBatchFactory.js

```diff
@@ -1,5 +1,5 @@
 import BaseFactory from './BaseFactory'
-import { dataToObjectById, groupIncludedByResource } from '@/api/JsonApi'
+import { dataToObjectById, groupIncludedByResource } from './../../src/api/JsonApi'
```

Comment:
Does @ not work?

Reply:
No, it won't work when factories are used in e2e tests. So we have used it this way across all factories.

Reply:
The @ is a vite / vue alias so is not available in the browser.